examples/plot_05_train_test_split.py

```diff
@@ -138,39 +143,48 @@
 X_train, X_test, y_train, y_test = skore.train_test_split(
```

Comment:
Line 140 "eventual" is not a good translation of "eventuellement" (false friend). Maybe "potential"?

Reply:
Done